### PR TITLE
Creating stored procedure for QUEUING_WAREHOUSES

### PR DESCRIPTION
**Pull request by DLampriadis**
## Summary
Checks for warehouses that have experienced queuing.

## Does this PR refer to any issue?
#10 

### Files
- queuing_warehouses.sql

### Procedure
- queuing_warehouses

### Tables
- queuing_warehouses_results

## I made sure that...
- [x] The files are named `{health_check_name}.sql/py`, are saved in the proper directory and are well documented
- [x] The procedure is named `{health_check_name}` and is stored in the schema `health_checks.public`
- [x] The tables created are named `{health_check_name}[_extra_info]_[results or intermediate]` and are stored in the schema `health_checks.public`
- [x] I run the pre-commit hooks in all my commits

## Does my code code need to be tested?
Yes

## The reviewer should check
1. Run procedure: call queuing_warehouses(); 
2. Select from created table: select * from health_checks.public.queuing_warehouses_results

### CODE REVIEW

File: snowflake-health-checks/src/libraries/procedures/queuing_warehouses.sql [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+;
+case
+    when
+        (
+            select max(total_avg_queued_load_seconds)
+            from health_checks.public.queuing_warehouses
+        )
+        > 1
+    then
+        return 'FAIL: There is at least a WAREHOUSE that the total average of queuing loads is higher than recommended. Check the warehouse(s) here: HEALTH_CHECKS.PUBLIC.QUEUING_WAREHOUSES'
```
Comment (EloiSanchez) [inline snowflake-health-checks/src/libraries/procedures/queuing_warehouses.sql:29]: The table to query is HEALTH_CHECKS.PUBLIC.QUEUING_WAREHOUSES_RESULTS. Also, use lowercase.

File: snowflake-health-checks/src/libraries/procedures/queuing_warehouses.sql [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+;
+else return 'PASS: All warehouses are running normally. There are not queuing loads.'
```
Comment (EloiSanchez) [inline snowflake-health-checks/src/libraries/procedures/queuing_warehouses.sql:31]: Add here the table to be queried with the results as well.

**Review by EloiSanchez — CHANGES_REQUESTED** [review]
All good, minor changes on the output of the results.

File: snowflake-health-checks/src/libraries/procedures/queuing_warehouses.sql (hunk 1 above)
Comment (wiesnseb) [inline snowflake-health-checks/src/libraries/procedures/queuing_warehouses.sql:29]: 'FAIL: There is at least one Warehouse with a total average of queuing loads higher than recommended. Check the warehouse(s) here: HEALTH_CHECKS.PUBLIC.QUEUING_WAREHOUSES'

File: snowflake-health-checks/src/libraries/procedures/queuing_warehouses.sql (hunk 2 above)
Comment (wiesnseb) [inline snowflake-health-checks/src/libraries/procedures/queuing_warehouses.sql:31]: 'PASS: All warehouses are running normally. There are no queuing loads'

**Comment by wiesnseb** [comment]
Can't checkout branch so couldn't run the code. Minor grammatical changes in the output

**Review by EloiSanchez — APPROVED** [review]
@DLampriadis check also the comments by seb about the grammar in the output. Feel free to change it or merge already.

**Comment by DLampriadis** [comment]
> @DLampriadis check also the comments by seb about the grammar in the output. Feel free to change it or merge already.

Yes, checked it and changed it now. I believe grammatically is correct now. If not please feel free to indicate where. Thank you

**Pull request merged**